layouts/_default/baseof.html

```diff
@@ -19,9 +19,12 @@
       <link rel="canonical" href="{{ .Permalink }}">
     {{ end }}
 
+    <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css?family=Lato:400,700%7CMerriweather:300,700%7CSource+Code+Pro:400,700&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fork-awesome/1.1.7/css/fork-awesome.min.css" integrity="sha512-9QjPqX/aCNwEQDyMqqMluNOSsHxTwOJNO3d4m5aUeNbyOPm8RcBA5hCUhvGmKFtSmQYGajqPopGtD60FWiWUwg==" crossorigin="anonymous" />
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" integrity="sha512-NhSC1YmyruXifcj/KFRWoC561YpHpc5Jtzgvbuzx5VozKpWvQ+4nXhPdFgmx8xqexRcpAglTj9sIBWINXa8x5w==" crossorigin="anonymous" />
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/fork-awesome@1.1.7/css/fork-awesome.min.css"
+      integrity="sha256-gsmEoJAws/Kd3CjuOQzLie5Q3yshhvmo7YNtBG7aaEY=" crossorigin="anonymous">
+    {{/* The minification is done by jsdelivr themselves and they donot recommend adding checksums for sutomatically minified files */}}
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/normalize.css@8/normalize.min.css">
```

Comment:
The version tag shouldn't be `2.3.1`?

Reply:
The latest version of fork-awesome is 1.1.7. https://forkaweso.me/Fork-Awesome/get-started/

Reply:
No, I'm talking about normalize. But got the wrong package, it should be `8.0.1`. I think we should use fixed version on all dependencies.

Reply:
Actually, the 8 there refers to 8.x.x. Since normalize.css is no longer maintained they will mean the same, at least for a long time.